examples/invaders/src/main.rs

```diff
@@ -165,6 +169,11 @@ fn main() -> Result<(), Error> {
                     return;
                 }
 
+                // Reset game
+                if g.game.input.key_pressed(VirtualKeyCode::R) {
```

Comment:
Please add a binding for controller inputs as well. Maybe the "select" button? I don't actually know what it's called any more on XBox/PS/Switch!

Reply:
Maybe the "select" button? I don't know！LOL ; )

But I believe that if add the selected operation interface, this problem will be solved!